### PR TITLE
DE-Translation: Revised strings 1 to 1000

### PR DESCRIPTION
**Pull request by violine1101**
Changed 'ss' to 'ß' where necessary, changed 'Kopieren' (Copy) to 'Klonen' (Clone) in several cases, translated 'Entity' to 'Objekt' (as it's the term in the wiki and in the official translation files) and some further overhaul

### CODE REVIEW

**Comment by Neui** [comment]
Hey, i just took a look. It's ok but there are some points: 
174: Why did you changed the - to a – character? Is it a special character to be used for listing?
344: You didn't translated it.. (Also, why got the translation removed? There was one)
373: The term "dump something" is used to say like "save something" - like you dump the memory, you save it (to your hard drive). You translated it as "entfernen" ("remove") wich is nos right.

**Comment by violine1101** [comment]
Thanks for your feedback. I assume you mean the translation codes (tXX and oXX). With "strings" in the title of this merge poll I actually meant the lines of the translation file.
* 174: That's an 'En dash'. It is used for subphrasing sentences – such as this one – and is a way to replace the (actually grammatically incorrect) minus (-). It's a bit longer but is not much different though. Can be changed back if you insist on the minus in the title, it was just an asthetic change.
* 344: I didn't remove the translation and I don't know why it is/was not there. I actually didn't know what that string actually is for because I couldn't find the string anywhere in MCEdit. (I changed the stars * to brackets so it's consistently.
* 373: Misunderstood dictionary. My bad.

**Comment by Neui** [comment]
* 174: Oh, that's nice to know.
* 344: Ok, i didn't expressed very good. I said there was a translation once (i translated it) but someone removed it. Under this list there is my version if you want to use it.
* 373: No problem.

My 344 translation if you want to bring it in:
```
t344       Rotiere die Position von deinem Händler
*Kann nur benutzt werden wenn "Unbeweglicher Dorfbewohner" angewählt ist*
```

**Comment by violine1101** [comment]
Can the "Trader" ("Händler") be something else than a Villager ("Dorfbewohner")? Currently, my translation is
````
Rotiere die Position des Dorfbewohners
(Kann nur benutzt werden, wenn er unbeweglich ist)
````
in English
````
Rotate the Villager's position
(Can only be used if he's unmovable)
````
My translation is shorter but also a bit inaccurate. I think it's okay though.

**Comment by TrazLander** [comment]
* 344 is in the "Create shops" filter.

File: lang/de_DE.trn [hunk 1]
```diff
@@ -17,59 +17,59 @@
...
 o18 Change Spawners While Nudging
-t18 Spawners ändern während des Schiebens
+t18 Spawner ändern beim Schieben
 o19 Change
 t19 Wechseln
 o20 Options
 t20 Optionen
 o21 OK
-t21 OK
+t21 Fertig
```
Comment (L3ice) [inline lang/de_DE.trn]: Ich hätte da jetzt lieber OK gelassen oder warum hast du es geändert?

File: lang/de_DE.trn [hunk 2]
```diff
@@ -691,21 +691,21 @@ t326 Geistlicher (violett)
...
 o329 Breeding Mode Ticks
-t329 Zucht Modus Ticks
+t329 Zuchtmodus-Ticks
 o330 Child/Adult Age
 t330 Kind/Erwachsener Alter
 o331 Repairs the backwards surfaces made by old versions of Minecraft.
 t331 Repariert die Oberflächen, die von alten Versionen von Minecraft erstellt wurden.
 o332 Makes water in the region flood outwards and downwards, becoming full source blocks in the process. This is similar to Minecraft Classic water.
-t332 Lässt das Wasser, die in der Region nach außen und nach unten fließen, zu volle Wasserblöcke werden. Das ist ähnlich wie Minecraft Classic Wasser.
+t332 Lässt Wasser in der Region nach außen unt nach unten fließen, wobei es zu vollen Wasserblöcken wird. Das ist dem Wasserverhaltens in Minecraft Classic ähnlich.
```
Comment (L3ice) [inline lang/de_DE.trn]: Da is dir noch ein kleiner Schreibfehler unterlaufen.

File: lang/de_DE.trn (hunk 1 above)
Comment (violine1101) [inline lang/de_DE.trn]: Im Spiel wurde es auch mit "Fertig" übersetzt. In MCEdit passt es aber nicht überall, ist mir beim weiteren Übersetzen aufgefallen. In der nächsten Version ändere ich das wieder, keine Sorge :)

File: lang/de_DE.trn (hunk 2 above)
Comment (violine1101) [inline lang/de_DE.trn]: Danke :D

**Comment by violine1101** [comment]
There is some update on my fork. I'll create another pull request again when I'm done.

**Pull request merged**